### ISSUE TITLE
Change functions to use HTTP.jl package instead of deprecated Requests and HttpCommon.

### ISSUE THREAD

**Issue by fernandogelin** [BioJulia/BioServices.jl#7]
## Expected Behavior
HTTP requests should be performed using the package HTTP.jl.

## Current Behavior
Currently, BioServices.jl uses Requests and HttpCommon which are deprecated.

## Implementation

Example of updated function:
```
import HTTP

function einfo(ctx::Associative=empty_context(); params...)
    params = process_parameters(params, ctx)
    return HTTP.request("GET", string(baseURL, "einfo.fcgi"), query=params)
end
```
Usage:
```
using BioServices.EUtils

res = einfo(db = "pubmed")

write("result.xml", res.body)

```

**Comment by fernandogelin** [comment]
I'll work on this issue and submit a pull request when I'm done.

**Comment by mirestrepo** [comment]
Solved by #8
